scheduler/scheduler.yml

```diff
@@ -66,6 +66,9 @@
   roles:
     - k8s_prepare_services
 
+- name: Check for Intel Gaudi accelerator
+  ansible.builtin.import_playbook: ../utils/check_intel_gaudi_device.yml
```

Comment:
Create role named as scheduler/roles/k8s_habana_container_runtime and place this task with file name check_pre_requisite.yml